openssl-sys/src/ocsp.rs

```diff
@@ -10,18 +10,18 @@ pub const OCSP_REVOKED_STATUS_CESSATIONOFOPERATION: c_int = 5;
 pub const OCSP_REVOKED_STATUS_CERTIFICATEHOLD: c_int = 6;
 pub const OCSP_REVOKED_STATUS_REMOVEFROMCRL: c_int = 8;
 
-pub const OCSP_NOCERTS: c_ulong = 0x1;
-pub const OCSP_NOINTERN: c_ulong = 0x2;
-pub const OCSP_NOSIGS: c_ulong = 0x4;
-pub const OCSP_NOCHAIN: c_ulong = 0x8;
-pub const OCSP_NOVERIFY: c_ulong = 0x10;
-pub const OCSP_NOEXPLICIT: c_ulong = 0x20;
-pub const OCSP_NOCASIGN: c_ulong = 0x40;
-pub const OCSP_NODELEGATED: c_ulong = 0x80;
-pub const OCSP_NOCHECKS: c_ulong = 0x100;
-pub const OCSP_TRUSTOTHER: c_ulong = 0x200;
-pub const OCSP_RESPID_KEY: c_ulong = 0x400;
-pub const OCSP_NOTIME: c_ulong = 0x800;
+pub const OCSP_NOCERTS: c_int = 0x1;
```

Comment:
Hmmm, kind of technically this is backwards incompatible I guess? Why is the change needed?

Reply:
Let me investigate further, I forgot about making this change. I think this is a difference in how bindgen is interpreting the preprocessor `#defines` in our sys crate and picking a c_int rather then a c_ulong. This is related to the comment further below as well.

Reply:
bindgen is right to choose a `c_int` here since these preprocessor constants are all representable as a `c_int` on almost all platforms (even if c_int happens to be `i16`), so their type in C is `c_int` (e.g. per the table in "The type of the integer constant" in [cppreference][1] - that's 6.4.4.1 in most C standards).

The `c_ulong` will have been chosen for simplicity since their primary use is as a flags argument to `OCSP_basic_verify()` (and a few other OCSP APIs that rust-openssl doesn't have bindings for).

[1]: https://en.cppreference.com/w/c/language/integer_constant